scripts/generate-report-pdf.py

```
@@ -0,0 +1,1138 @@
+    )
+    parser.add_argument("--csv", help="Read from local CSV instead of VPS")
+    parser.add_argument("--epoch-json", help="Epoch JSON file (offline mode)")
+    parser.add_argument("--output", help="Output PDF file path")
+    parser.add_argument("--html-output", help="Optional output path for rendered HTML")
+    parser.add_argument("--date", help="Date label for report title")
+    parser.add_argument(
+        "--dry-run", action="store_true", help="Print summary only (no PDF generation)"
+    )
+    args = parser.parse_args()
+
+    date_label = args.date or datetime.now().strftime("%B %d, %Y")
```

Ruff (0.15.5)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)